tools/nxstyle.c

```diff
@@ -477,7 +543,7 @@ int main(int argc, char **argv, char **envp)
           (strcmp(line, " * Private Functions\n") == 0 ||
            strcmp(line, " * Public Functions\n") == 0))
         {
-          //fprintf(stderr, "Functions begin at line %d:%d\n", lineno, n);
```

Comment:
Commented out code not permitted.  Use of C++ // commenting not permitted.  Coding standard only permit #if 0 for commented out code WITH an short explanation for why is is commented out.

Otherwise, commented out code should be removed.